code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -504,7 +503,7 @@
 			var/mob/living/critter/flock/F = owner
 			if(F)
 				F.tri_message("<span class='notice'>[owner] begins forming a cuboid structure around [target].</span>",
-					F, "<span class='notice'>You begin imprisoning [target]. You will both need to stay still for this to work.</span>",
```

Comment:
Change back to original thing (I think since new comment doesn't apply anymore)?

Reply:
Done!